tbx/project_styleguide/templates/patterns/atoms/date_and_reading_time/date_and_reading_time.html

```diff
@@ -0,0 +1,9 @@
+{% if date %}
+    <time datetime="{{ date|date:'Y-m-d' }}">{{ date }}</time>
```

Comment:
This might need updating based on the changes in https://github.com/torchbox/torchbox.com/pull/83/files

Reply:
Yes I'll rebase if necessary, but there are some questions over that MR at the moment